internal/pkg/plan/persist.go

```diff
@@ -23,22 +24,19 @@ type persistPlanBuilder struct {
 }
 
 func (b *persistPlanBuilder) build() {
-	// New configs
+	// Process children of the existing objects
 	for _, path := range b.UntrackedDirs() {
```

Comment:
Pred zmenou: 
- jeden `for` pre hladanie `configs` v `branch`
- a dalsi for nizsie, pre hladanie `rows` v configu
- okrem toho, ak sa nasiel novy config, tak sa dalsi `for` spustal tam, na hladanie `rows`